javascript/ql/src/semmle/javascript/HtmlSanitizers.qll

```diff
@@ -48,6 +48,11 @@ private class DefaultHtmlSanitizerCall extends HtmlSanitizerCall {
       or
       callee = LodashUnderscore::member("escape")
       or
+      exists(DataFlow::PropRead read | read = callee |
+        read.getPropertyName() = "sanitize" and
+        read.getBase().asExpr().(VarAccess).getName() = "DOMPurify"
```

Comment:
Could we base this on `moduleImport` and `globalVarRef` instead?

Reply:
That could work, but only if we type-tracked the `moduleImport`.   
(This is where DOMPurify is required: https://github.com/mycolorway/simditor/blob/ef01a643cbb7f8163535d6bfb71135f80ec6a6fd/lib/simditor.js#L16). 

So for now, I think I prefer to keep it as is.

Reply:
Good point. I've added a note about it in https://github.com/github/codeql-javascript-team/issues/118#issuecomment-633989671